frontend/src/i18n/locales/zh-CN.json

```diff
@@ -301,7 +301,9 @@
   },
   "AccountSent": {
     "connect-wallet": "连接钱包",
-    "connect-your-wallet": "连接您的钱包以发送资金"
+    "connect-your-wallet": "连接您的钱包以发送资金",
+    "need-signature": "此应用程序需要您的签名才能解密本地存储的发送历史记录",
+    "sign": "标志"
```

Comment:
@garyghayrat can you confirm these? Note my suggestion in the above comment when checking the translations